Wordle.Api/Wordle.Api/Identity/IdentitySeed.cs

```diff
@@ -43,5 +50,24 @@ private static async Task SeedAdminUserAsync(UserManager<AppUser> userManager)
                 await userManager.AddToRoleAsync(user, Roles.Admin);
             }
         }
+
+        if (await userManager.FindByEmailAsync("Awesome@intellitect.com") == null)
+        {
```

Comment:
This looks great, is a little funny that the admin account has less permissions than the other, but functionally wise it all looks great.

Reply:
Yah, I'm probably going to update this maybe if we decide to keep working on it past the end of this class.